src/scss/main.scss

```diff
@@ -583,48 +679,70 @@ $curve-easeOutBack: cubic-bezier(0.175, 0.885, 0.32, 1.275);
   }
 }
 
-.braintree-method__check-container {
-  position: relative;
+/* =========================================================================
+   Dropin Visibility States
+   ========================================================================= */
+.braintree-show-card .braintree-card,
+.braintree-show-methods .braintree-methods,
+.braintree-show-options .braintree-options,
+.braintree-show-paypal .braintree-paypal {
+  display: block;
+  height: auto;
+  overflow: visible;
+  visibility: visible;
+}
 
-  &:before {
-    content: '';
-    position: absolute;
-    width: 41px;
-    height: 41px;
-    background: $background-color;
-    border-radius: 100px;
-    transform: scale(0.9);
-  }
+[data-braintree-id='methods-label'],
+[data-braintree-id='choose-a-way-to-pay'] {
+  position: absolute;
+  transform: translateY(30px);
+  transition: transform 200ms ease;
+  visibility: hidden;
 }
 
-.braintree-methods {
-  display: none;
-  margin: 0 auto 13px auto;
+.braintree-show-methods [data-braintree-id='methods-label'],
+.braintree-show-options [data-braintree-id='choose-a-way-to-pay'] {
+  transform: none;
+  visibility: visible;
 }
 
-.braintree-methods .braintree-methods,
-.braintree-options .braintree-options,
-.braintree-paypal .braintree-paypal,
-.braintree-card .braintree-card {
-  display: block;
+.braintree-show-methods.braintree-show-options [data-braintree-id='choose-a-way-to-pay'] {
+  transform: translateY(30px);
+  visibility: hidden;
 }
 
-[data-braintree-id="other-ways-to-pay"] {
-  display: none;
+/* =========================================================================
+   Javascript hooks
+   ========================================================================= */
+.braintree-loaded [data-braintree-id='methods'],
+.braintree-loaded [data-braintree-id='options'] {
+  max-height: 1000px; // Must set specific value here in order to have animation
+  opacity: 1;
+  transition: max-height 250ms $curve-easeOutBack, opacity 300ms linear 100ms;
+  min-height: 0;
 }
 
-[data-braintree-id="choose-a-way-to-pay"] {
-  display: block;
+.braintree-show-methods,
+.braintree-show-options {
+  [data-braintree-id='other-ways-to-pay'] {
+    display: none;
+  }
 }
 
-.braintree-methods [data-braintree-id="other-ways-to-pay"],
-.braintree-paypal [data-braintree-id="other-ways-to-pay"],
-.braintree-card [data-braintree-id="other-ways-to-pay"] {
+.braintree-show-card [data-braintree-id='other-ways-to-pay'],
+.braintree-show-methods [data-braintree-id='other-ways-to-pay'],
+.braintree-show-paypal [data-braintree-id='other-ways-to-pay'] {
   display: block;
 }
 
-.braintree-methods [data-braintree-id="choose-a-way-to-pay"],
-.braintree-paypal [data-braintree-id="choose-a-way-to-pay"],
-.braintree-card [data-braintree-id="choose-a-way-to-pay"] {
+/* =========================================================================
+   Remove this when we have a payment methods width 100%  width    
```

Comment:
I'm a little confused about if this is styling added after a transition or if it's stuff we want to remove later